src/index.html

```diff
@@ -4,9 +4,243 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="page__header">
+      <a class="page__header-logo" href="/"><img src="/images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="is-active nav__link" href="#">apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">смартфоны и телефоны</a></li>
+          <li class="nav__item" data-qa="nav-hover"><a class="nav__link" href="#">ноутбуки и компьютеры</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">гаджеты</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">планшеты</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">фото</a></li>
+          <li class="nav__item"><a class="nav__link" href="#">видео</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page__content">
+      <div class="page__card" data-qa="card">
+        <img class="page__card-image"
+             src="/images/imac.jpeg"
+             alt="imac"
+        >
+        <h1 class="page__card-title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+        <h2 class="page__card-info">Код товара: 195434</h2>
+        <section class="page__section">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="page__section-rating">Отзывов: 5</div>
+        </section>
+        <section class="page__section">
+          <div class="page__section-text">Цена:</div>
+          <div class="page__section-value">69 999 грн</div>
```

Comment:
```suggestion
          <p class="page__section-text">Цена:</p>
          <p class="page__section-value">69 999 грн</p>
```
Це текстова інформація. Краще використовувати підходящі для цього теги